samples/deployment-manager/deploy-forseti.yaml.sample

```diff
@@ -17,13 +17,13 @@
 
 # Change these import paths as necessary to suit your environement.
 imports:
-- path: py/inventory/cloudsql-instance.py
+- path: ../../deployment-templates/inventory/cloudsql-instance.py
```

Comment:
QQ: What is the consideration to use relative paths like this?  How about absolute path?  Would something like this work?
```
<root path>/deployment-templates/inventory/cloudsql-instance.py
```